apps/aecore/lib/aecore/txs/pool/worker.ex

```diff
@@ -78,7 +76,9 @@ defmodule Aecore.Txs.Pool.Worker do
           Logger.info("Transaction is already in pool")
         else
           # Broadcasting notifications for new transaction in a pool(per account and every)
-          Notify.broadcast_new_transaction_in_the_pool(tx)
+          if match?(%SpendTx{}, tx.data) do
+            Notify.broadcast_new_transaction_in_the_pool(tx)
```

Comment:
Why do we only broadcast SpendTx-s?

Reply:
(This limits the chance any oracle tx will be minded in a big network significantly.)

Reply:
oh noticed this as well, but forgot to comment

Reply:
This only excludes the transaction from websocket notifications, not  peer broadcast

Reply:
So then this should be fixed if "comment" `apps/aehttpserver/lib/aehttpserver/web/channels/notify.ex` line 17 should be fixed.

Reply:
this is still a valid comment for @d-velev to address